src/leveled_sst.erl

```diff
@@ -2164,8 +2164,11 @@ check_blocks([Pos|Rest], BlockPointer, BlockLengths, PosBinLength,
                     PosBinLength,
                     BlockNumber,
                     additional_offset(IdxModDate)),
-    R = fetchfrom_rawblock(BlockPos, deserialise_block(BlockBin, PressMethod)),
-    case {R, LedgerKeyToCheck} of
+    Pid =
+        spawn(
+            ?MODULE, check_block, [self(), BlockPos, BlockBin, PressMethod]),
+    Result = receive {checked_block, Pid, R} -> R end,
```

Comment:
This is just code to verify an idea... otherwise, we should have used some more caution here on timeout of a possible crash of spawned code, etc.... but not for this experiment.